classes/Langchecker/DotLangParser.php

```diff
@@ -78,20 +86,25 @@ function ($tag) {
                 }
 
                 // Get file description
-                if (Utils::startsWith($current_line, '## NOTE:')) {
+                if (! isset($dotlang_data['filedescription'])
```

Comment:
We support multiple ```## NOTE:##``` to avoid having huge lines of text (example in screenshots), so we shouldn't check if ```filedescription``` is already set.